src/timebar/charts/vessel-events.js

```diff
@@ -179,10 +202,8 @@ class VesselEvents extends Component {
       >
         {ICONS[highlightedEvent.type]}
         <div className={styles.tooltipText}>
-          <div className={styles.tooltipDate}>
-            {dayjs(highlightedEvent.start).format('MMM D YYYY')}
-          </div>
-          {title}
+          <div className={styles.tooltipDate}>{start.format('MMM D YYYY h:mm a')}</div>
```

Comment:
Worth it to move this to a const ? or even to a prop with a default value to make it configurable depending on the app `'MMM D YYYY h:mm a'`